falcon/request.py

```diff
@@ -514,6 +516,55 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    @property
+    def access_route(self):
+        """A list of all addresses from client to the last proxy server.
+
+        Inspired by werkzeug's `access_route`. Notice this list may contain
+        string(s) other than IPv4 / IPv6 address. For example the "unknown"
+        identifier and obfuscated identifier defined by RFC 7239.
+
+        Warning: HTTP Forwarded headers can be forged by any client or proxy.
```

Comment:
So that sphinx will highlight this paragraph as a warning, can you indent the paragraph, e.g:

```
Warning:
    HTTP Forwarded headers can be forged by any client or proxy.
    Use this property with caution and...
```

https://gist.github.com/kgriffs/05667fb34fb3b69b8030